### PR TITLE
Update 1-TwoSum.java

### PR DESCRIPTION
**Pull request by abc-ahsan**

### CODE REVIEW

File: Java/1-TwoSum.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
-
+
```
Comment (touqeer-ahmad) [inline Java/1-TwoSum.java:2]: I don't think, we should have a space here

**Review by touqeer-ahmad — CHANGES_REQUESTED** [review]

File: Java/1-TwoSum.java (hunk 1 above)
Comment (abc-ahsan) [reply Java/1-TwoSum.java:2]: Yeah Think so. But You have to think about it... Please Don't ignore....

Comment (touqeer-ahmad) [reply Java/1-TwoSum.java:2]: Did you test it out in the QA environment

Comment (abc-ahsan) [reply Java/1-TwoSum.java:2]: Yup.. and it works Perfect...

Comment (touqeer-ahmad) [reply Java/1-TwoSum.java:2]: Perfect, :)

**Review by touqeer-ahmad — APPROVED** [review]

**Pull request merged**